vipps-recurring-api.md

```diff
@@ -537,36 +566,110 @@ when the product is shipped.
+| `event campaign`      | A set price for until a given date with a text description for a given event                             | 20kr until Christmas and then 100kr each month                                          |
+| `full flex campaign`  | A set price for a different interval until a given date                                                  | 10kr per week until 2022-05-01T00:00:00Z and then 200kr a month                         |
 
-In order to start a campaign the campaign field has to be added either to the agreement draft
-[`POST:/recurring/v2/agreements`][draft-agreement-endpoint]
-for a campaign in the start of an agreement or update an agreement
-[`PATCH:/recurring/v2/agreements/{agreementId}`](https://vippsas.github.io/vipps-recurring-api/#/Agreement%20Endpoints/updateAgreement)
-for an ongoing agreement. When adding a campaign
-while drafting a new agreement the start date is ignored and the current
-date-time is used. All dates must be in date-time format as according to
-[RFC-3999](https://www.ietf.org/rfc/rfc3339.txt).
+In order to start a campaign the campaign field has to be added to the agreement draft
+[`POST:/recurring/v3/agreements`][draft-agreement-endpoint]
+for a campaign in the start of an agreement
 
+**Price campaign**
 ```json
 {
   "campaign": {
-    "start": "2019-05-01T00:00:00Z",
-    "end": "2019-06-01T00:00:00Z",
-    "campaignPrice": 49900
+    "campaignType": "PRICE_CAMPAIGN",
+    "end": "2022-06-01T00:00:00Z",
+    "price": 49900
+  }
+}
+```
+| Field               | Description                                                                 |
+|---------------------|-----------------------------------------------------------------------------|
+| `campaignType`      | The type of the campaign                                                    |
+| `price`             | The price that the customer will pay for each interval during the campaign  |
+| `end`               | The end date of the campaign                                                |
+
+TODO add screenshot
+
+**Period campaign**
+```json
+{
+ "campaign": {
+    "type": "PERIOD_CAMPAIGN",
+    "price": 100,
+    "period": {
+      "unit": "WEEK",
+      "count": 2
+    }
   }
 }
 ```
+| Field          | Description                                                                    |
+|----------------|--------------------------------------------------------------------------------|
+| `campaignType` | The type of the campaign                                                       |
+| `price`        | The price that the customer will pay for the period of the campaign            |
+| `period.unit`  | The period where the campaign price is applied. Can be DAY, WEEK, MONTH, YEAR  |
+| `period.count` | The number of periods the campaign should run for                              |
```

Comment:
```suggestion
| `period`  | The period where the campaign price is applied. Consist of a Unit and a Count, example; { "unit": "WEEK", "count": 4 }  |
```

I think the explanation is very clunky. Period is a data structure used elsewhere and shouldn't need re-explanation everywhere